docs/developer/api-reference/mutations/checkout-complete.mdx

```diff
@@ -32,6 +33,10 @@ Client-side generated data required to finalize the payment.
 
 URL of a view where users should be redirected to see the order details. URL in RFC 1808 format.
 
+#### `paymentId` ([`ID`](../../../developer/api-reference/scalars/id))
+
+New in Saleor 3.0. The ID of the payment used to finalize the checkout. Mandatory for checkouts with multiple payments.
```

Comment:
`Mandatory for checkouts with multiple payments.` Do we require a paymentID in case when someone will process all payments with `checkoutPaymentComplete`?

Reply:
It looks so, check it out [here](https://github.com/saleor/saleor/blob/feature/partial-payments-3.0/saleor/graphql/payment/mutations.py#L262)